src/main/java/emissary/config/ServiceConfigGuide.java

```diff
@@ -862,7 +862,39 @@ public Map<String, Set<String>> findStringMatchMultiMap(@Nullable final String p
      */
     @Override
     public Map<String, Set<String>> findStringMatchMultiMap(@Nullable final String param, final boolean preserveOrder) {
+        if (param == null) {
+            return Map.of();
+        }
 
+        return findStringMatchMultiMap(param, false, preserveOrder);
+    }
+
+    /**
+     * Find entries beginning with the specified string and return a hash keyed on the remainder of the string with the
+     * value of the config line as the value of the hash Multiple values for the same hash are allowed and returned as a
+     * Set.
```

Comment:
nit-pick here, assuming the formatting moved this from two lines to one at `Multiple values`. otherwise looks good to me